src/widgets/ActiveField.php

```
@@ -962,4 +975,53 @@ protected function addErrorClassIfNeeded(&$options)
+
+            if ($tag === false) {
+                $this->parts['{label}'] = $options['label'];
+            } elseif ($tag === 'label') {
+                $this->parts['{label}'] = Html::activeLabel(
+                    $this->model,
+                    $this->attribute,
+                    [
+                        'label' => $options['label'],
+                        ...$this->labelOptions,
+                    ],
+                );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii\helpers\Html' in method 'generateLabel'. (undefined)

(StaticAccess)